settings.toml

```diff
@@ -6,6 +6,7 @@ support_info = "Ping {} for any questions"
 rust_version_info = "latest nightly from rustup"
 hw_info = "Benchmarks are run in a controlled sandbox with limited resources."
 methodology = "We use criterion to benchmark. Submissions may be manually reviewed."
+#bot_token = "DISCORD_TOKEN_HERE"
```

Comment:
I'm going to have to stomp on this, but I also didn't document/explain.

You're supposed to make your own `.secrets.toml` file in the root of the repo, and put the bot_token there. This helps reduce the chance of sensitive config data from getting checked in.

There's a few other things that go in there, so let me write up a README quick with some notes on how to run the bot, hopefully that'll help.

And yes, we should have a clearer error when you try to run the bot without a token. That's [now an issue](https://github.com/proegssilb/ferris-elf/issues/22).

Reply:
Sure see the latest commit. I also added the config validation here since that's a pretty small change.